docs/src/componentDocs/UserMenu/examples/UserMenuWithCustomHeader.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { CodeBlock, CodeBlockActionButtonRow } from '../../../shared';
+import * as colors from '@brightlayer-ui/colors';
+import { UserMenuWithCustomHeaderExample } from './UserMenuWithCustomHeaderExample';
+
+const codeSnippet = `<UserMenu
+    onClick={(): void => setOpen(!open)}
+    avatar={<Avatar src={tRex} alt={'User Avatar'} />}
```

Comment:
@bkarambe , can we add a string here? Right now it is showing file name.
<img width="562" alt="Screenshot 2022-09-27 at 11 14 32 AM" src="https://user-images.githubusercontent.com/10433274/192442433-e3f27ba6-7396-432c-8407-85dfdd9f87d3.png">